tests/Unit/Classes/ToolsTest.php

```diff
@@ -470,11 +474,11 @@ public function providerMathRound(): array
     }
 
     /**
-     * @dataProvider providerMathRound
+     * @dataProvider providerPsRound
      */
-    public function testMathRound(float $expectedResult, float $value, int $precision, int $mode): void
+    public function testPsRound(float $expectedResult, float $value, int $precision, int $mode): void
     {
-        $this->assertSame($expectedResult, Tools::math_round($value, $precision, $mode));
+        $this->assertSame($expectedResult, Tools::ps_round($value, $precision, $mode));
```

Comment:
```suggestion
        $this->assertSame($expectedResult, Tools::ps_round($value, $precision, $mode));
        // This method is deprecated and will be removed in V10, we just keep the tests to avoid regressions
        $this->assertSame($expectedResult, Tools::math_round($value, $precision, $mode));
```
Maybe we should keep both checks until it is fully removed

Reply:
You can't, because math_round doesn't support PS_ROUND_UP and PS_ROUND_DOWN

Reply:
I will make it call ps_round. :-)